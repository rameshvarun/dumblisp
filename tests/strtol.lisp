(print (strtol "3213"))
(print (strtol "0xF10c"))
