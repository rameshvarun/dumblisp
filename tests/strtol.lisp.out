3213
61708
